Source/Core/VideoCommon/FreeLookCamera.cpp

```diff
@@ -234,10 +234,26 @@ void FreeLookCamera::Reset()
   m_dirty = true;
 }
 
+void FreeLookCamera::ModifySpeed(float multiplier)
+{
+  m_speed *= multiplier;
+}
+
+void FreeLookCamera::ResetSpeed()
+{
+  m_speed = 1.0f;
+}
+
+float FreeLookCamera::GetSpeed() const
+{
+  return m_speed;
+}
+
 void FreeLookCamera::DoState(PointerWrap& p)
 {
   if (p.mode == PointerWrap::MODE_WRITE || p.mode == PointerWrap::MODE_MEASURE)
   {
+    p.Do(m_speed);
```

Comment:
You should update the state version over here since you changed this:
https://github.com/dolphin-emu/dolphin/blob/0ef9b998cea75ee7025b2446f7c70f8efa296dd5/Source/Core/Core/State.cpp#L76-L77

Reply:
Ah good call.  Thanks for reminding me